phys/module_cumulus_driver.F

```diff
@@ -1317,6 +1344,17 @@ SUBROUTINE cumulus_driver(grid                                     &
                ,do_capsuppress=do_capsuppress                   &
                ,cap_suppress_loc=cap_suppress_loc               &
 #endif 
+!++ GF CTRAN ++ lyy 01/2020
```

Comment:
go ahead and follow the spacing of the "&" character in the surrounding code

Reply:
Hi Dave, I have finished making changes according to your comments. But I find that I cannot modify this pr now. Can I just create another pr to release-4.2?